service/routes.py

```diff
@@ -244,6 +244,11 @@ class AddressCollection(Resource):
     #------------------------------------------------------------------
     # CREATE A NEW ADDRESS FOR THE CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('create_addresses')
+    @api.response(404, 'Customer not found')
+    @api.response(400, 'The posted Address data was not valid')
+    @api.expect(create_address_model)
+    @api.marshal_with(create_address_model)  
```

Comment:
I think that we would want address_model here instead of create_address_model, since we'll want customer and address id's to be returned

Reply:
yeah, I just changed :)